sdk/tests/client/signing/alias.rs

```diff
@@ -51,8 +51,8 @@ async fn sign_alias_state_transition() -> Result<()> {
         1_000_000,
         alias_id,
         0,
-        bech32_address_0,
-        bech32_address_1,
+        &bech32_address_0.to_string(),
+        &bech32_address_1.to_string(),
```

Comment:
Since this became the norm, maybe these constructors should take a Bech32Address and internally convert. I can do it if you want.